src/Symfony/Component/Serializer/Command/DebugCommand.php

```diff
@@ -99,13 +99,20 @@ private function getAttributesData(ClassMetadataInterface $classMetadata): array
 
         foreach ($classMetadata->getAttributesMetadata() as $attributeMetadata) {
             $data[$attributeMetadata->getName()] = [
-                'groups' => $attributeMetadata->getGroups(),
-                'maxDepth' => $attributeMetadata->getMaxDepth(),
-                'serializedName' => $attributeMetadata->getSerializedName(),
-                'ignore' => $attributeMetadata->isIgnored(),
                 'normalizationContexts' => $attributeMetadata->getNormalizationContexts(),
                 'denormalizationContexts' => $attributeMetadata->getDenormalizationContexts(),
+                'groups' => $attributeMetadata->getGroups(),
+                'ignore' => $attributeMetadata->isIgnored(),
+                'maxDepth' => $attributeMetadata->getMaxDepth(),
```

Comment:
Can we avoid changing the order here?

Reply:
I wanted to keep the order at first as well and had two possible solutions.

The first one: 

```php
foreach ($classMetadata->getAttributesMetadata() as $attributeMetadata) {
    $data[$attributeMetadata->getName()]['groups'] = $attributeMetadata->getGroups();
    $data[$attributeMetadata->getName()]['maxDepth'] = $attributeMetadata->getMaxDepth();
    
    if ($serializedName = $attributeMetadata->getSerializedName()) {
        $data[$attributeMetadata->getName()]['serializedName'] = $serializedName;
    }

    if ($serializedPath = $attributeMetadata->getSerializedPath()) {
        $data[$attributeMetadata->getName()]['serializedPath'] = (string) $serializedPath;
    }
    
    $data[$attributeMetadata->getName()]['ignore'] = $attributeMetadata->isIgnored();
    $data[$attributeMetadata->getName()]['normalizationContexts'] = $attributeMetadata->getNormalizationContexts();
    $data[$attributeMetadata->getName()]['denormalizationContexts'] = $attributeMetadata->getDenormalizationContexts();
}
```

and the second one:

```php
foreach ($classMetadata->getAttributesMetadata() as $attributeMetadata) {
    $serializedNameOrPathKey = 'serializedName';
    $serializedNameOrPathValue = null;
    if ($serializedName = $attributeMetadata->getSerializedName()) {
        $serializedNameOrPathValue = $serializedName;
    }

    if ($serializedPath = $attributeMetadata->getSerializedPath()) {
        $serializedNameOrPathKey = 'serializedPath';
        $serializedNameOrPathValue = (string) $serializedPath;
    }

    $data[$attributeMetadata->getName()] = [
        'groups' => $attributeMetadata->getGroups(),
        'maxDepth' => $attributeMetadata->getMaxDepth(),
        $serializedNameOrPathKey => $serializedNameOrPathValue,
        'ignore' => $attributeMetadata->isIgnored(),
        'normalizationContexts' => $attributeMetadata->getNormalizationContexts(),
        'denormalizationContexts' => $attributeMetadata->getDenormalizationContexts(),
    ];
}
```

I didn't really like either one of them and decided to just order the keys alphabetically.

We could also always show both `serializedName` and `serializedPath`, even though one of them is always `null`. WDYT?

Reply:
I just changed it to always output `serializedName` and `serializedPath`.